src/store/models/taro.ts

```diff
@@ -54,6 +74,64 @@ const taroModel: TaroModel = {
     const balances = await api.listBalances(node);
     actions.setBalances({ node, balances });
   }),
+  getAllInfo: thunk(async (actions, node) => {
+    await actions.getAssets(node);
+    await actions.getBalances(node);
+  }),
+
+  mintAsset: thunk(
+    async (actions, payload, { injections, getStoreState, getStoreActions }) => {
+      const {
+        node,
+        assetType,
+        name,
+        amount,
+        metaData,
+        enableEmission,
+        skipBatch,
+        autoFund,
+      } = payload;
+      const TaroNode = node as TarodNode;
```

Comment:
The type of `node` is already `TarodNode` so this statement is redundant.

Reply:
Done